src/main/java/org/alliancegenome/curation_api/model/entities/base/AuditedObject.java

```diff
@@ -71,14 +71,10 @@ public class AuditedObject extends BaseEntity {
 	@Column(columnDefinition = "boolean default false", nullable = false)
 	private Boolean obsolete = false;
 
-	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
-	@KeywordField(name = "dbDateCreated_keyword", sortable = Sortable.YES, searchable = Searchable.YES, aggregable = Aggregable.YES, valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
 	@JsonView({ View.FieldsOnly.class })
 	@CreationTimestamp
 	private OffsetDateTime dbDateCreated;
 
-	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
-	@KeywordField(name = "dbDateUpdated_keyword", sortable = Sortable.YES, searchable = Searchable.YES, aggregable = Aggregable.YES, valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
```

Comment:
Dropping these from the index?

Reply:
I was running into issues with the limit on the number of indexed fields (1,000).  I figured we don't need these indexed.

Reply:
We can change that in the definition for the index.

Reply:
here is the setting: `index.mapping.total_fields.limit` it goes in the: `src/main/resources/hibernate/custom-index-settings.json` file.

Reply:
@oblodgett - Done